TGEngine/io/Resource.cpp

```diff
@@ -78,27 +88,50 @@ void loadResourceFile(const char* name, Map* map) {
 
 		// Reads the indices from the file
 		actorInfo.pIndices = new uint32_t[actorInfo.indexCount]; // Object lifetime ?
-		fread(&actorInfo.pIndices, sizeof(uint32_t), actorInfo.indexCount, file);
+		fread(actorInfo.pIndices, sizeof(uint32_t), actorInfo.indexCount, file);
 
 		// calculates the length of the vertices to read
-		uint32_t len = blocklength - (sizeof(ActorInputInfo) + sizeof(uint32_t) * actorInfo.indexCount);
+		uint32_t len = blocklength - (74 + sizeof(uint32_t) * actorInfo.indexCount);
```

Comment:
> honestly, having specific hardcoded numbers is not good programming practice and both of us know it, so i'd say we should make the 74 ~~static class constant in the actor class~~ constant defined somewhere

Reply:
I agree with that tho, I am proposing a Macro

Reply:
I have an even better idea: We can just make this whole calculation obsolete